jsoc/gsoc/medpipe3d.md

```diff
@@ -0,0 +1,82 @@
+# MedPipe3D Projects 
+
+MedPipe3D.jl together with MedEye3D.jl MedEval3D.jl and currently in development MedImage.jl is a set of libraries created to provide essential tools for 3D medical imaging to the Julia language ecosystem. Recently all of the projects migrated also to the JuliaHealth organization to improve packages visibility and promote interoperability.
+
+
+## Potential Projects
+
+### Project 1: Adding functionalities to medical imaging visualizations
+
+**Description:** 
+MedEye3D is a package that supports the display of medical imaging data. It includes multiple functionalities specific to this use case like automatic windowing to display soft tissues, lungs, and other tissues. The display takes into account voxel spacing, support of overlaying display for multimodal imaging, and more. All with high performance powered by OpenGL and Rocket.jl. Still, a lot of further improvements are possible and are described in the Potential Outcomes section. 
+
+- **Mentor:** Jakub Mitura [email: jakub.mitura14@gmail.com]
+
+- **Difficulty**: Hard
+
+- **Duration**: 350 hours
+
+- **Suggested Skills and Background**: 
+  - Experience with Julia
+  - Basic familiarity with computer graphics preferably OpenGL 
+  - Some experience with 3d volumetric data with spatial metadata (or a willingness to learn!)

```

Comment:
Just a suggestion, but in the past, students have found it helpful when I list what packages to be familiar with as well as any links to requisite background knowledge (youtube videos, articles, etc.). Might want to consider adding them in.